common/djangoapps/third_party_auth/saml.py

```diff
@@ -286,22 +286,22 @@ def generate_bizx_oauth_api_saml_assertion(self, user_id):
         """
         session = requests.Session()
         transaction_data = {
-            'operation_name': 'generate_bizx_oauth_api_saml_assertion',
-            'endpoint_url': self.sapsf_idp_url,
             'token_url': self.sapsf_token_url,
-            'company_id': self.odata_company_id,
```

Comment:
No need to have this stuff in the POST anymore?

Reply:
Right -- `operation_name` and `endpoint_url` are keys that we've introduced (ie, they are not SF API params), and `company_id` is not required for this particular transaction, we are only including it in order to provide the context in the exception log.